website/static/js/registrationUtils.js

```diff
@@ -44,6 +45,8 @@ function Comment(data) {
         self.lastModified = new Date();
     });
 
+    self.created = data.created || new Date();
```

Comment:
@samchrisinger what do you mean? Should `data.created` become `new Date(data.created)`?